core/src/main/java/org/apache/accumulo/core/logging/TabletLogger.java

```diff
@@ -121,6 +121,10 @@ private static Collection<String> asFileNames(Collection<CompactableFile> files)
     return Collections2.transform(files, CompactableFile::getFileName);
   }
 
+  public static void renamed(KeyExtent extent, TabletFile src, TabletFile dest) {
+    fileLog.debug("{} renamed {} to {}", extent, src.getFileName(), dest.getFileName());
```

Comment:
I added this as debug but it might be noisy enough to be dropped to trace.
It's a pretty minimal message
`2025-11-25T18:11:23,346 121 [tablet.files] DEBUG: 1<< renamed F0000001.rf_tmp to F0000001.rf`